### PR TITLE
misleading message when class not found

### PR DESCRIPTION
**Pull request by rosiecki**

### CODE REVIEW

**Comment by deki** [comment]
Thanks for the pull request. 
What is shown before and after the patch?

**Comment by rosiecki** [comment]
I'm assuming we have a webservice:
```
package xxx.yyy.zzz;
@WebService(endpointInterface="xxx.yyy.zzz.WsEndpointInterface")
public class WsImpl {
}
```
- Message before patch: "Unable to load the class xxx.yyy.zzz.WsImpl"
- Message after patch: "Unable to load the class xxx.yyy.zzz.WsEndpointInterface"

File: rt/frontend/jaxws/src/main/java/org/apache/cxf/jaxws/ServiceImpl.java [hunk 1]
```diff
@@ -589,8 +589,7 @@ private QName getPortTypeName(Class<?> serviceEndpointInterface) {
                 try {
                     seiClass = Thread.currentThread().getContextClassLoader().loadClass(epi);
                 } catch (ClassNotFoundException e) {
-                    Message msg = new Message("COULD_NOT_LOAD_CLASS", BUNDLE,
-                                              seiClass.getCanonicalName());
+                    Message msg = new Message("COULD_NOT_LOAD_CLASS", BUNDLE, api);
```
Comment (deki) [inline rt/frontend/jaxws/src/main/java/org/apache/cxf/jaxws/ServiceImpl.java]: Should be epi and not api

Comment (rosiecki) [inline rt/frontend/jaxws/src/main/java/org/apache/cxf/jaxws/ServiceImpl.java]: I fixed it.

**Comment by deki** [comment]
I've applied your changes. Thank you!

**Pull request closed without merging**